src/appimagetool.c

```diff
@@ -578,7 +578,7 @@ main (int argc, char *argv[])
     GOptionContext *context;
 
     // initialize help text of argument
-    sprintf(_exclude_file_desc, "Uses given file as exclude file for mksquashfs, in addition to %s.", APPIMAGEIGNORE);
+    snprintf(_exclude_file_desc, "Uses given file as exclude file for mksquashfs, in addition to %s.", APPIMAGEIGNORE);
```

Comment:
You need to specify a size here properly. The compiler implicitly casts the string to an int here.